src/EditorFeatures/CSharp/Outlining/CSharpOutliningService.cs

```diff
@@ -40,6 +40,7 @@ internal class CSharpOutliningService : AbstractOutliningService
             builder.Add<MethodDeclarationSyntax, MethodDeclarationOutliner, MetadataAsSource.MethodDeclarationOutliner>();
             builder.Add<NamespaceDeclarationSyntax, NamespaceDeclarationOutliner>();
             builder.Add<OperatorDeclarationSyntax, OperatorDeclarationOutliner, MetadataAsSource.OperatorDeclarationOutliner>();
+            builder.Add<ParenthesizedLambdaExpressionSyntax, ParenthesizedLambdaExpressionOutliner>();
```

Comment:
Correct. I tried to explain that to you at one point when I lamented them not working for parenthesized lambdas, but I don't think I was clear. We also have lots of unit tests for the parenthesized lambda outliner. It just wasn't being composed.